README.md

```diff
@@ -10,21 +10,24 @@ We are calling those new way of packaging Deep Learning models: __Rockets__.
 __Welcome to the Rockets Scientists Community!!!__
 
 ## Install the repositories
-We recommend you to use an isolated Python environement such as [virtualenv](https://virtualenv.pypa.io/en/latest/) or [conda](https://docs.conda.io/en/latest/) with at least __Python 3.5__. Then you can use the following lines of code:
+We recommend you to use an isolated Python environement such as [virtualenv](https://virtualenv.pypa.io/en/latest/) or [conda](https://docs.conda.io/en/latest/) with at least __Python 3.6__. Then you can use the following lines of code:
```

Comment:
Why did you change it to python 3.6? The default python version on most system is still 3.5. And it should still work with 3.5.

Reply:
officially, the PIP package only supports python 3.6+

However, on some systems, the code might also work on lower python versions.